app/assets/javascripts/components/ops/tenant-component.js

```diff
@@ -15,6 +15,11 @@ tenantFormController.$inject = ['API', 'miqService'];
 function tenantFormController(API, miqService) {
   var vm = this;
 
+  vm.modelName = 'tenantModel';
+  vm.saveable = miqService.saveable;
+  vm.resetClicked = miqService.resetClicked;
+  vm.cancelClicked = miqService.cancelClicked;
+
```

Comment:
@mtho11 @himdel This almost feels like boilerplate code that would have to be repeated in every component.
And, matter of time before Code Climate hits us with `Similar code found in 1 other location` - which is the same redundancy issue that I'm trying to resolve.